RELEASE_NOTES.md

```diff
@@ -58,6 +58,8 @@ This release adds support to pass `None` values via channels and revamps the `Ti
   Previously, only the file path was reported. With this update, users can now determine if a file change is a creation, modification, or deletion.
   Note that this change may require updates to your existing code that relies on `FileWatcher` as the new interface returns a `FileWatcher.Event` instead of just the file path.
 
+* `FileWatcher` now accepts a`collections.abc.Iterable` object as `event_types` argument instead of a `set` or `None`, and defaults to a `frozenset` containing all available event types to ensure a consistent default behavior while enabling customizations.
```

Comment:
> Only the None change should be mentioned here, as it is the only change affecting the user. The change from set to Iterable is transparent for the user, or should be mentioned in the New Features as now more types can be used as an argument, but users don't need to change existing code.

Right, will update it

> Any reason not to take an Iterator instead of an Iterable? This would make it even more flexible (although in practice I doubt it makes any difference).

I've just tried it and `mypy` is not happy with it as it cannot assign the default  `frozenset` due to incompatibilities. See the error message below

```sh
nox > mypy --namespace-packages --non-interactive --install-types --explicit-package-bases --strict -p benchmarks -p docs -p frequenz.channels -p tests
src/frequenz/channels/util/_file_watcher.py:43: error: Incompatible default for argument "event_types" (default has type "FrozenSet[EventType]", argument has type "Iterator[EventType]")  [assignment]
src/frequenz/channels/util/_file_watcher.py:43: note: "frozenset" is missing following "Iterator" protocol member:
src/frequenz/channels/util/_file_watcher.py:43: note:     __next__
```

Reply:
Updated release notes